tests/test2.cpp

```diff
@@ -27,10 +27,10 @@ SafetyHookMid g_hook{};
 int main() {
     std::cout << add_42(2) << "\n";
 
-    // Lets disassemble add_42 and hook its RET.
+    // Let's disassemble add_42 and hook its RET.
     ZydisDecoder decoder{};
 
-#if defined(_M_X64)
+#ifdef _M_X64
     ZydisDecoderInit(&decoder, ZYDIS_MACHINE_MODE_LONG_64, ZYDIS_STACK_WIDTH_64);
 #elif defined(_M_IX86)
```

Comment:
I prefer:

```c++
#if defined(_M_X64)
...
#elif defined(_M_IX86)
...
#endif
```

Over:

```c++
#ifdef _M_X64
...
#elif defined(_M_IX86)
...
#endif
```

Because it's more consistent.

Reply:
I agree. I'll fix all of these. Once MSVC adds `elifdef` then it'll be a good time to switch.